package.json

```diff
@@ -118,11 +116,18 @@
   },
   "browserslist": [
     "last 1 Chrome version",
-    "last 1 Android version",
     "last 1 Firefox version",
-    "last 1 FirefoxAndroid version",
-    "last 1 Safari version",
-    "last 1 iOS version",
     "last 1 Edge version"
-  ]
+  ],
+  "release-it": {
+    "github": {
+      "release": false
```

Comment:
I think `false` is the default so this can be removed if this isn't a wanted feature.